LX.StaffScheduler.Api/appsettings.Development.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=HOME-PC;Initial Catalog=CafeDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=RAPKUPICH;Initial Catalog=CafeDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```

Comment:
can be omitted toward to leave these settings of connection not modified toward to exclude the json file from diff